easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2.2-foss-2018b-Python-3.6.6.eb

```diff
@@ -18,9 +19,7 @@ source_urls = ['https://github.com/s-will/%(name)s/releases/download/v%(version)
 sources = [SOURCELOWER_TAR_GZ]
 checksums = ['76024e6d12a52e36a2d1aaffb09a7e7c1148ade05e7584530e21ce584c26e45b']
 
-dependencies = [
-    ('ViennaRNA', '2.4.11', '-Python-3.6.6'),
-]
+dependencies = [('ViennaRNA', '2.4.11', '-Python-3.6.6')]
```

Comment:
@SethosII Avoid hardcoding the `Python` version in multiple places, just use `versionsuffix` here:

```python
dependencies = [('ViennaRNA', '2.4.11', versionsuffix)]